apps/crn-server/src/data-providers/contentful/analytics.data-provider.ts

```diff
@@ -207,17 +210,29 @@ export class AnalyticsContentfulDataProvider implements AnalyticsDataProvider {
                 ...((team.inactiveSince &&
                   currentWorkingGroupIdsFromTeamLeaders) ||
                   []),
-                ...previousWorkingGroupIdsFromTeamLeaders,
+                ...removeDuplicates(
+                  previousWorkingGroupIdsFromTeamLeaders,
+                  currentWorkingGroupIdsFromTeamLeaders,
+                ),
               ]),
               workingGroupMemberCount: team.inactiveSince
                 ? 0
-                : getUniqueIdCount(currentWorkingGroupIdsFromTeamMembers),
+                : getUniqueIdCount([
+                    ...currentWorkingGroupIdsFromTeamLeaders,
+                    ...currentWorkingGroupIdsFromTeamMembers,
+                  ]),
 
               workingGroupPreviousMemberCount: getUniqueIdCount([
                 ...((team.inactiveSince &&
                   currentWorkingGroupIdsFromTeamMembers) ||
                   []),
-                ...previousWorkingGroupIdsFromTeamMembers,
+                ...removeDuplicates(
+                  [
+                    ...previousWorkingGroupIdsFromTeamMembers,
+                    ...previousWorkingGroupIdsFromTeamLeaders,
+                  ],
+                  currentWorkingGroupIdsFromTeamMembers,
```

Comment:
I was trying to think about an example and I came up with this

![working-groups](https://github.com/yldio/asap-hub/assets/16595804/8efafdc7-1eb9-447d-9f74-5c8ccb0eb5f8)

In this case above, the `workingGroupPreviousMemberCount` should be 0 because there isn't a working group where all the users associated with this team are past members, right? But it's giving 1